cypress/testcases/virtualmachines/advanced.spec.ts

```diff
@@ -78,6 +82,43 @@ describe("Create a new VM and add Install guest agent option", () => {
     cy.get('.tab#advanced').click()
     vms.usbTablet().expectChecked()
 
-    vms.delete(VM_NAME)
+    vms.deleteProgramlly(`${NAMESPACE}/${VM_NAME}`)
   })
 })
+
+describe("Verify Booting in EFI mode checkbox", () => {
+  beforeEach(() => {
+    login.login();
+  });
+
+  it('Create a new VM with Booting in EFI mode checked', () => {
+    const VM_NAME = 'test-efi'
+    const NAMESPACE = 'default'
+
+    const value = {
+      name: VM_NAME,
+      cpu: '2',
+      memory: '4',
+      image: 'ubuntu-18.04-server-cloudimg-amd64.img',
```

Comment:
Is there a reason we're using an old version of ubuntu instead of the LTS or openSUSE Leap?

Reply:
Changed the image name as env variable, now can configure in `cypress.env.json` depending on the real environment. Feel free to change the default value in `cypress.json`.